website/routes.py

```diff
@@ -227,6 +228,20 @@ def make_url_map(app):
         ),
 
         Rule('/news/', 'get', {}, OsfWebRenderer('public/pages/news.mako')),
+
+        Rule(
+            '/prereg/',
+            'get',
+            prereg.prereg_landing_page,
+            OsfWebRenderer('prereg_landing_page.mako')
```

Comment:
Please add the `trust=False` flag to mako routes, to ensure that markupsafe rendering is turned on.